CHANGELOG.md

```diff
@@ -23,7 +23,10 @@ following procedure to upgrade all the nodes running a head:
 Only when this procedure has been applied to all Hydra nodes can you open a new
 head again.
 
-- Fuel removal: 
+- **BREAKING** Hydra TUI now uses `--cardanoSigningKey` flag to specify a user
+  key used to select UTxO and submit a commit transaction.
```

Comment:
The option is called `--cardano-signing-key`

This is not strictly a breaking change, but can keep it one. Users most likely need to change the invocation of the `hydra-tui` to _not_ point to the internal wallet key.

If it makes things clearer, I'm also fine in renaming the option.. i.e. artifically making this a really  breaking change with an error message shown to the user. Then, they would also re-read the new options `--help` message.